recipes/mss/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = "1.2.0" %}
+
+package:
+  name: mss
+  version:  {{ version }}
+
+source:
+    fn: mss-{{ version }}.tar.gz
+    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
+    sha256: 6e15a2a7c65ce676169551660e51ced5b0718ef83c4f93104a0f6b70ed502183
+
+build:
+    number: 0
+    script: python setup.py install
+    entry_points:
+      - mss = mslib.msui.mss_pyui:main
+      - mswms = mslib.mswms.mswms:main
+
+requirements:
+  build:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl >=7.45.0
+    - cycler >=0.10.0
+    - execnet >=1.3.0
+    - freetype >=2.5.5
+    - functools32 >=3.2.3.2
+    - hdf5 >=1.8.15.1
+    - libpng >=1.6.22
+    - lxml >=3.6.0
+    - matplotlib >=1.5.1
+    - netcdf4 >=1.2.2
```

Comment:
Some of your dependencies are over specified b/c `netcdf4` will install `hdf5`, so there is no need to add `hdf5` to the recipe.